### ISSUE TITLE
About the ‘MS’ features

### ISSUE THREAD

**Issue by MagicXY** [MAZiqing/FEDformer#15]
Thank you for your wonderful work. But i have a little question about the features set as ‘MS’. In the source code of exp_main.py :

f_dim = -1 if self.args.features == 'MS' else 0     
                    batch_y = batch_y[:, -self.args.pred_len:, f_dim:].to(self.device) 
                    loss = criterion(outputs, batch_y)    
                    train_loss.append(loss.item())
My question is that how to promise the dimensions of  "outputs" is same as "batch_y". It's really confuse me for a couple days. 
Thanks

**Comment by tianzhou2011** [comment]
Checkout the code in data_loader.py, you should be able to figure it out.  M/MS/S are just 3 different tasks.
